### ISSUE TITLE
Indiana Jones and The Last Crusade (1989)

### ISSUE THREAD

**Issue by Eulisker** [exoscoriae/eXoDOS#4050]
There is a rolling demo of this game with some (very slight) unique content. At least another version of the intro. There might be some more differences to the original game (I can not remember the surprise Hitler in Castle Brunwald) but i am not that familiar with the EGA version.

[indy3ega.zip](https://github.com/exoscoriae/eXoDOS/files/9127165/indy3ega.zip)

This is not in the main game for sure

![image](https://user-images.githubusercontent.com/81642149/179387924-6b9b8449-53e4-42c7-82e9-6903e5c671b2.png)

**Comment by exoscoriae** [comment]
added to the main game entry. that said, this should not have been filed under manuals/extras

**Comment by Eulisker** [comment]
I scanned

- [ ] Red Lens
- [ ] Translation Table
- [ ] Translation Table (revealed)

The current translation table looks like it is from a budget release. 
I don't know if you want to put both Translation Tables in. I like it because it gives you an idea how the copy protection worked but it is not needed.

Added to the github folder.

**Comment by exoscoriae** [comment]
replaced old one. added

**Comment by Eulisker** [comment]
I found

- [ ] Manual EGA
- [ ] Reference Card

The manual for the EGA version is slightly different to the current VGA one. The text is laid out differently, there are different screenshots, there is some info about the copy protection in the end and the ad on the back page is different. Also all the information in the reference card is put into the VGA manual. There are probably more differences but that was what i noticed. I think it is different enough to have this as an extra.

Added to the github folder.
